src/appengine/handlers/cron/load_bigquery_stats.py

```diff
@@ -140,21 +119,24 @@ def _load_data(self, bigquery, fuzzer):
       else:
         schema = kind.SCHEMA
 
-      self._update_schema_if_needed(bigquery, dataset_id, table_id, schema)
-
       gcs_path = fuzzer_stats.get_gcs_stats_path(kind_name, fuzzer, timestamp)
+      load = {
+          'destinationTable': {
+              'projectId': project_id,
+              'tableId': table_id + '$' + date_string,
+              'datasetId': dataset_id,
+          },
+          'schemaUpdateOptions': ['ALLOW_FIELD_ADDITION',],
```

Comment:
Note that this is intentionally added, otherwise, new fields don't get accepted.